custom/templates/DefaultRevamp/register.tpl

```diff
@@ -4,7 +4,9 @@
 <h2 class="ui header">
     {$CREATE_AN_ACCOUNT}
     {if $OAUTH_FLOW}
-    <div class="sub header">{$OAUTH_MESSAGE_CONTINUE}</div>
+    <div class="sub header">
+        {$OAUTH_MESSAGE_CONTINUE} <a href="{$OAUTH_CANCEL_REGISTER_URL}">({$CANCEL})</a>
```

Comment:
Does hardcoding parentheses here work for all languages?

Reply:
Good question, I'll have to see

Reply:
No longer an issue